ya2d/PSPBUILD

```diff
@@ -0,0 +1,28 @@
+pkgname=ya2d
+pkgver=20251217
+pkgrel=3
+pkgdesc=" Yet Another 2D library (for PSP) "
```

Comment:
Small change in formatting
```suggestion
pkgdesc="Yet Another 2D library (for PSP)"
```

Reply:
I copy-pasted from yaml-cpp so it will need fixing there too, do I do it here in this PR?